keyboards/herringbone/keymaps/iso/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
```

Comment:
I guess this file was supposed to be `keyboards/herringbone/keymaps/via/rules.mk`?